src/Plutarch/Extra/Value.hs

```diff
@@ -1,6 +1,12 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE OverloadedRecordDot #-}
```

Comment:
Do we need these extensions? I'm particularly wary of `DuplicateRecordFields` if it can at all be avoided.